neuralforecast/models/timellm.py

```diff
@@ -15,7 +15,17 @@
 from ..losses.pytorch import MAE
 
 try:
-    from transformers import GPT2Config, GPT2Model, GPT2Tokenizer
+    from transformers import (
```

Comment:
This fails if any of the imports fail. Why not only load GPT2?

Reply:
I’ve updated it to load the default model(gpt2) if it fails to fetch.